R/navs-legacy.R

```diff
@@ -186,41 +405,47 @@ navs_bar_ <- function(..., title = NULL, id = NULL, selected = NULL,
 #  (with minor modifications)
 # -----------------------------------------------------------------------
 
-navbarPage_ <- function(title,
-                       ...,
-                       id = NULL,
-                       selected = NULL,
-                       sidebar = NULL,
-                       fillable = TRUE,
-                       gap = NULL,
-                       padding = NULL,
-                       position = c("static-top", "fixed-top", "fixed-bottom"),
-                       header = NULL,
-                       footer = NULL,
-                       inverse = FALSE,
-                       underline = TRUE,
-                       collapsible = FALSE,
-                       fluid = TRUE,
-                       theme = NULL) {
-
+navbarPage_ <- function(
+  title,
+  ...,
+  id = NULL,
+  selected = NULL,
+  sidebar = NULL,
+  fillable = TRUE,
+  gap = NULL,
+  padding = NULL,
+  position = c("static-top", "fixed-top", "fixed-bottom"),
+  header = NULL,
+  footer = NULL,
+  inverse = FALSE,
+  underline = TRUE,
+  collapsible = FALSE,
+  fluid = TRUE,
+  theme = NULL
+) {
```

Comment:
For git history and minimal diff reasons, I'd prefer to keep the "legacy" code changes to only meaningful changes

Reply:
Yeah good point, I thought I was going to need to modify these. Hang on, I'll get them out of the diff